File: src/main/java/lach_01298/moreBees/ClientProxy.java
package lach_01298.moreBees;

import net.minecraft.client.renderer.block.model.ModelResourceLocation;
import net.minecraft.item.Item;
import net.minecraft.world.World;
import net.minecraftforge.client.MinecraftForgeClient;
import net.minecraftforge.client.model.ModelLoader;

public class ClientProxy extends CommonProxy {
	public void registerProxies() 
	{
	}
	@Override
	public void registerItemRenderer(Item item, int meta, String id) 
	{
		ModelLoader.setCustomModelResourceLocation(item, meta, new ModelResourceLocation(MoreBees.MOD_ID + ":" + id, "inventory"));
	}

}

File: src/main/java/com/tencao/morebees/proxy/ClientProxy.kt
package com.tencao.morebees.proxy

import com.tencao.morebees.MBCore
import net.minecraft.client.renderer.block.model.ModelResourceLocation
import net.minecraft.item.Item
import net.minecraftforge.client.model.ModelLoader

internal class ClientProxy : CommonProxy() {

    override fun registerItemRenderer(item: Item, meta: Int, id: String) {
        ModelLoader.setCustomModelResourceLocation(item, meta, ModelResourceLocation(MBCore.MODID + ":" + id, "inventory"))
    }
}